internal/connect/connection.go

```diff
@@ -81,7 +83,33 @@ func successCode(code int) bool {
 	return code >= 200 && code < 300
 }
 
+// Returns true if proxy setup is enabled at the system level. This is specific
+// to SUSE.
+func proxyEnabled() bool {
+	value := strings.TrimSpace(os.Getenv("PROXY_ENABLED"))
+
+	// If the environment variable is not set, ignore all this and let Go figure
+	// the rest of it.
+	if value == "" {
+		return true
+	}
+
+	// If the parsing fails, let's assume that the value is bad and ignore it.
+	v, err := strconv.ParseBool(value)
```

Comment:
https://pkg.go.dev/strconv#ParseBool is different to https://github.com/SUSE/connect/commit/b2758adacf2092cbbef8b8286c201a7d1ba5e7e1#diff-925e25ec51fea79a5b120af303ad78fd1f34220e30134129edaa4bd73c77ff8dR86 

What exactly does yast use for true/false?

Reply:
Fixed